Data/Scripts/016_UI/017_UI_PokemonStorage.rb

```diff
@@ -34,13 +34,20 @@ def createFusionIcon(species)
     icon1 = AnimatedBitmap.new(GameData::Species.icon_filename(headPoke))
     icon2 = AnimatedBitmap.new(GameData::Species.icon_filename(bodyPoke))
 
+    dexNum = getDexNumberForSpecies(species)
```

Comment:
```
    directory_name = "Graphics/Pokemon/FusionIcons"
    Dir.mkdir(directory_name) unless File.exists?(directory_name)
```
needs to be added, to check if directory exist, and create it if it doesn't exist. Game will crash without this code if the dir doesn't exist !